vendor/router-resource.js

```diff
@@ -1,5 +1,5 @@
 (function() {
```

Comment:
I had to change the below resource function from https://github.com/emberjs/ember.js/pull/15883/files#diff-57d1dbf7bb82d085435a5e830a2e77c7. We should move back to the original default arguments